docker/bash/config.sh

```diff
@@ -6,8 +6,10 @@
 #### Global Variables:
 COMPOSE_PROJECT_NAME=pc
 COMPOSE_DIR="${DOCKER_DIR}/compose"
-COMPOSE_FILES=$(find ${COMPOSE_DIR} -not -path "${COMPOSE_DIR}/OS/*" -not -path "${COMPOSE_DIR}/CI/*" -name "*.yml")
+# Use the new stack.yml with includes, plus OS and CI specific files
+COMPOSE_FILES="${COMPOSE_DIR}/fca-low/stack.yml"
 
+# OS specific overrides (e.g., macOS Docker socket permissions)
 COMPOSE_DIR_OS_SPECIFIC="${COMPOSE_DIR}/OS/$(uname -s)"
 COMPOSE_FILES_OS_SPECIFIC=
 
```

Comment:
I just tested the stack on macOS with the current Docker Desktop without loading the macOS-specific override (`docker/compose/OS/Darwin/shared.yml`) and the stack seems to work fine (`docker-gen` properly reacts to container creations and deletions to update the nginx configuration).

So I suggest getting rid of the OS-specific override mechanism.

Reply:
Covered by #160 .